src/pages/Account/Login.tsx

```diff
@@ -17,20 +27,47 @@ import "./Login.scss";
 
 export interface LoginProps {
   formType?: "initial" | "login" | "register" | "passwordReset";
-  venue: AnyVenue;
+  venue: WithId<AnyVenue>;
 }
 
 export const Login: React.FC<LoginProps> = ({
   formType = "initial",
   venue,
 }) => {
+  const venueId = venue.id;
+
   const [formToDisplay, setFormToDisplay] = useState(formType);
 
   const { signInWithSAML, hasSamlAuthProviderId } = useSAMLSignIn(
     venue.samlAuthProviderId
   );
-  // It will be extended with addition of new providers
-  const hasAlternativeLogins = hasSamlAuthProviderId;
```

Comment:
Why was it removed? What if there are no authConfig/saml integrations? We don't want to show the `Quick login block`

Reply:
Look at line 70 where it still exists, but without the comment which is rendundant now.